pkg/issuer/acme/BUILD.bazel

```diff
@@ -10,42 +10,27 @@ go_library(
     visibility = ["//visibility:public"],
     deps = [
         "//pkg/acme:go_default_library",
+        "//pkg/acme/accounts:go_default_library",
         "//pkg/acme/client:go_default_library",
         "//pkg/api/util:go_default_library",
         "//pkg/apis/certmanager/v1alpha2:go_default_library",
         "//pkg/apis/meta/v1:go_default_library",
-        "//pkg/client/listers/acme/v1alpha2:go_default_library",
         "//pkg/controller:go_default_library",
         "//pkg/issuer:go_default_library",
         "//pkg/logs:go_default_library",
         "//pkg/util/errors:go_default_library",
+        "//pkg/util/kube:go_default_library",
         "//pkg/util/pki:go_default_library",
         "@io_k8s_api//core/v1:go_default_library",
         "@io_k8s_apimachinery//pkg/api/errors:go_default_library",
         "@io_k8s_apimachinery//pkg/apis/meta/v1:go_default_library",
+        "@io_k8s_client_go//kubernetes/typed/core/v1:go_default_library",
         "@io_k8s_client_go//listers/core/v1:go_default_library",
-        "@io_k8s_utils//clock:go_default_library",
+        "@io_k8s_client_go//tools/record:go_default_library",
         "@org_golang_x_crypto//acme:go_default_library",
     ],
 )
 
-go_test(
```

Comment:
These tests never did anything and this package is not covered by unit tests. Decided to remove altogether for now instead of kidding ourselves :)